configs/application/config.json

```diff
@@ -217,4 +217,4 @@
 		"$applicationRoot/configs/application/services.json",
 		"$applicationRoot/configs/application/securityPolicies.json"
 	]
-}
\ No newline at end of file
+}
```

Comment:
Probably should drop this change, since it isn't needed and just makes the diff more complicated.

Reply:
will do.